test/vcf/assembly_checker_integration_test.cpp

```diff
@@ -72,40 +86,56 @@ namespace ebi
         }
 
         SECTION("Multiple entry, multiple mismatch")
         {
             auto folder = boost::filesystem::path("test/input_files/v4.3/assembly_checker/failed/failed_multiplemismatch_multipleentry/");
             std::string file_prefix = folder.parent_path().filename().string();
-            std::ifstream vcf_input{folder.string() + file_prefix + ebi::vcf::VCF_EXT};
+            std::string vcf_path = folder.string() + file_prefix + ebi::vcf::VCF_EXT;
+            std::ifstream vcf_input{vcf_path};
             std::ifstream fasta_input{folder.string() + file_prefix + ebi::vcf::FASTA_EXT};
             std::ifstream fasta_index_input{folder.string() + file_prefix + ebi::vcf::FASTA_EXT + ebi::vcf::INDEX_EXT};
-            CHECK_FALSE(ebi::vcf::assembly_checker::check_vcf_ref(vcf_input, fasta_input, fasta_index_input, outputs));
+            CHECK_FALSE(ebi::vcf::assembly_checker::check_vcf_ref(vcf_input, vcf_path, fasta_input, fasta_index_input, outputs));
         }
 
         SECTION("Empty FASTA file, single entry")
         {
             auto folder = boost::filesystem::path("test/input_files/v4.3/assembly_checker/failed/failed_empty_fasta/");
             std::string file_prefix = folder.parent_path().filename().string();
-            std::ifstream vcf_input{folder.string() + file_prefix + ebi::vcf::VCF_EXT};
+            std::string vcf_path = folder.string() + file_prefix + ebi::vcf::VCF_EXT;
+            std::ifstream vcf_input{vcf_path};
             std::ifstream fasta_input{folder.string() + file_prefix + ebi::vcf::FASTA_EXT};
             std::ifstream fasta_index_input{folder.string() + file_prefix + ebi::vcf::FASTA_EXT + ebi::vcf::INDEX_EXT};
-            CHECK_FALSE(ebi::vcf::assembly_checker::check_vcf_ref(vcf_input, fasta_input, fasta_index_input, outputs));
+            CHECK_FALSE(ebi::vcf::assembly_checker::check_vcf_ref(vcf_input, vcf_path, fasta_input, fasta_index_input, outputs));
         }
 
         SECTION("Multiple entry, all mismatch")
         {
             auto folder = boost::filesystem::path("test/input_files/v4.3/assembly_checker/failed/failed_allmismatch_multipleentry/");
             std::string file_prefix = folder.parent_path().filename().string();
-            std::ifstream vcf_input{folder.string() + file_prefix + ebi::vcf::VCF_EXT};
+            std::string vcf_path = folder.string() + file_prefix + ebi::vcf::VCF_EXT;
+            std::ifstream vcf_input{vcf_path};
             std::ifstream fasta_input{folder.string() + file_prefix + ebi::vcf::FASTA_EXT};
             std::ifstream fasta_index_input{folder.string() + file_prefix + ebi::vcf::FASTA_EXT + ebi::vcf::INDEX_EXT};
-            CHECK_FALSE(ebi::vcf::assembly_checker::check_vcf_ref(vcf_input, fasta_input, fasta_index_input, outputs));
+            CHECK_FALSE(ebi::vcf::assembly_checker::check_vcf_ref(vcf_input, vcf_path, fasta_input, fasta_index_input, outputs));
         }
+
+        // TODO: make this work in windows
```

Comment:
We could wrap this block with a macro so it is still run on Linux and MacOS, as described [here](https://stackoverflow.com/a/8249232).